docker-compose.yaml

```diff
@@ -1,43 +1,41 @@
-version: "3.8"
+version: '3.8'
 services:
-  db:
-    container_name: postgres
-    image: postgres
-    restart: always
-    healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U admin -d rps"]
-      interval: 3s
-      retries: 5
-      start_period: 30s
+  mongodb:
+    container_name: mymongo
+    image: mongo
+    restart: unless-stopped
     env_file:
-      - postgres.env
+      - mongo.env
     volumes:
-      - postgres-data:/var/lib/postgresql/data
-    ports:
-      - 5432:5432
+      - ./.docker/mongodb/data/db/:/data/db/
+      - ./.docker/mongodb/data/log/:/var/log/mongodb/
     networks:
       - backend-network
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=MONGO_USER
```

Comment:
I believe these environment variables should go in the `mongo.env` file, that's the whole point of using a `.env` file after all, so you don't have to store usernames and passwords in plaintext in a docker compose file which is committed to version control for everyone to see

Reply:
according to @Rebecca-CYLi this was throwing some errors so I'm just testing this out now